### PR TITLE
Add wasm-pack test to github workflows

### PR DESCRIPTION
**Pull request by farmaazon**
### Pull Request Description
To the test workflow added a step where wasm-pack test is run for each crate in workspace.

### Important Notes
- A script was added, which runs wasm-pack test for each workspace members, because The wasm-pack itself cannot read workspaces. The script was written in rust, therefore a special crate for build-and-utility-scripts in rust was created.
- The wasm32 target does not support -Zprofile flag, therefore wasm_bindgen_tests cannot work with the used coverage reports generators. For now I left it as it is, in future maybe someone find another tool supporting wasm32.

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/luna/enso/blob/master/doc/rust-style-guide.md), [Scala](https://github.com/luna/enso/blob/master/doc/scala-style-guide.md), [Java](https://github.com/luna/enso/blob/master/doc/java-style-guide.md) or [Haskell](https://github.com/luna/enso/blob/master/doc/haskell-style-guide.md) style guides as appropriate.
- [x] All code has been tested where possible.

### CODE REVIEW

File: script/rust/src/bin/test-all.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+}
+
+/// Call wasm-pack test for each workspace member
+///
+/// This function reads workspace members list from `Cargo.toml` in current
+/// directory, and call `wasm-pack test` each member. All script arguments
+/// are passed to wasm-pack test process.
+fn main()
+{
```
Comment (wdanilo) [inline script/rust/src/bin/test-all.rs]: formatting

File: lib/system/web/js/resize_observer.js [hunk 2]
```diff
@@ -54,12 +54,12 @@ class Pool {
...
 let resizeObserverPool = new Pool((...args) => new ResizeObserver(...args))
 
-export function resize_observe(target, f) {
+function resize_observe(target, f) {
     let id = resizeObserverPool.reserve(resize_observer_update(f))
     resizeObserverPool[id].observe(target)
 }
 
-export function resize_unobserve(id) {
+function resize_unobserve(id) {
```
Comment (wdanilo) [inline lib/system/web/js/resize_observer.js]: Why those changes?

File: script/rust/src/bin/test-all.rs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+fn get_workspace_members(
+    cargo_toml_root : toml::Value
+) -> Vec<String> {
```
Comment (wdanilo) [inline script/rust/src/bin/test-all.rs]:
```rust
fn get_workspace_members
(cargo_toml_root : toml::Value) -> Vec<String> {
```

File: lib/system/web/js/resize_observer.js (hunk 2 above)
Comment (farmaazon) [reply lib/system/web/js/resize_observer.js]: Once I changed test in core from #[test] to #[wasm_bindgen_test] and it turns out that wasm-pack test does not work with "export" word (even with --node option...). But I don't know why, I don't have a deep knowledge of Js.

Comment (farmaazon) [reply lib/system/web/js/resize_observer.js]: The exact error is ```Running /home/adam-praca/dev/basegl/target/wasm32-unknown-unknown/debug/deps/basegl-c02b39c2bb2e07d5.wasm
/home/adam-praca/dev/basegl/target/wasm32-unknown-unknown/wbg-tmp/snippets/basegl-system-web-056d2c0780aef067/js/resize_observer.js:57
export function resize_observe(target, f) {
^^^^^^

SyntaxError: Unexpected token 'export'
```
But without exports the example 01-scene does not work. Now I'm putting version which works on both browser and wasm-pack test

File: script/rust/src/bin/test-all.rs (hunk 1 above)
Comment (farmaazon) [reply script/rust/src/bin/test-all.rs]: fixed

File: script/rust/src/bin/test-all.rs (hunk 3 above)
Comment (farmaazon) [reply script/rust/src/bin/test-all.rs]: fixed

File: lib/system/web/js/resize_observer.js (hunk 2 above)
Comment (wdanilo) [reply lib/system/web/js/resize_observer.js]: strange :/

**Pull request merged**